### PR TITLE
Add documentation for new ecobee.set_sensors_in_climate Service

### PR DESCRIPTION
**Pull request by myztillx**
## Proposed change

Add documentation to coincide with changes proposed in: https://github.com/home-assistant/core/pull/102871

The change is to add a service which allows the setting of participating sensors for a thermostat and climate program.

This PR replaces PR https://github.com/home-assistant/home-assistant.io/pull/29560 which was closed since it targeted the wrong branch.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102871
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new service action for the Ecobee integration allowing users to set active sensors for specific climate programs.
	- Updated documentation to include details on the new service and its required attributes.

### CODE REVIEW

File: source/_integrations/ecobee.markdown [hunk 1]
```diff
@@ -241,3 +242,14 @@ Enable/disable Smart Home/Away and Follow Me modes.
...
+
+### Service `ecobee.set_sensors_in_climate`
+
+Set which sensors are active on a termostat for a specific climate program.
+
+| Service data attribute | Optional | Description                                                                                     |
+| ---------------------- | -------- | ----------------------------------------------------------------------------------------------- |
+| `entity_id`            | no       | ecobee thermostat on which to set the active sensors                                            |
+| `climate_name`         | yes      | Name of the climate program to set the sensors active on (defaults to currently active program) |
+| `sensors`              | no       | Sensors to set as participating for climate.                                                    |
```
Comment (emontnemery) [inline source/_integrations/ecobee.markdown]: Please double check to make sure this matches the attribute names in the PR

Comment (myztillx) [reply source/_integrations/ecobee.markdown]: You were right. I had changed `climate_name` to `preset_mode` and did not update here. Updated.

File: source/_integrations/ecobee.markdown [hunk 2]
```diff
@@ -241,3 +242,14 @@ Enable/disable Smart Home/Away and Follow Me modes.
...
+
+### Service `ecobee.set_sensors_in_climate`
+
+Set which sensors are active on a termostat for a specific climate program.
+
+| Service data attribute | Optional | Description                                                                                                                                         |
+| ---------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `entity_id`            | no       | ecobee thermostat on which to set the active sensors                                                                                                |
+| `preset_mode`          | yes      | Name of the climate program to set the sensors active on (defaults to currently active program)                                                     |
+| `sensors`              | no       | Sensors to set as participating for climate. This is the device id of the sensor/thermostat. These can be found in the available_sensors attribute. |
```
Comment (c0ffeeca7) [inline source/_integrations/ecobee.markdown]:
```suggestion
| `entity_id`            | no       | ecobee thermostat on which to set the active sensors.                                                                                                |
| `preset_mode`          | yes      | Name of the climate program to set the sensors active on (defaults to currently active program).                                                     |
| `sensors`              | no       | Sensors to set as participating for climate. This is the device ID of the sensor/thermostat. These can be found in the available_sensors attribute. |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @myztillx :+1:

**Pull request merged**